### ISSUE TITLE
Dialog content not announced by voiceover on chrome

### ISSUE THREAD

**Issue by DBD324** [mi6/ic-design-system#617]
## Summary of the bug

When using voiceover on chrome, dialog content is not announced when launched. 

## 🪜 How to reproduce

Tell us the steps to reproduce the problem:

1. Go to page: Dialogs
2. Open voiceover
3. Navigate to 'launch dialog' button in component demo and click it. 
4. See dialog open and announcement 'dialog, Confirm, button.'

Problem only seems to occur using voiceover on chrome. 

Using voiceover on safari, or NVDA on windows seems to announce the contents fine. 

## 📸 Screenshots or code

If you can, add screenshots to show the problem. If you need to show a code example, you can include a snippet or link to a repository with an example.

## 🖥 📱 Device

- Type: Desktop
- Device: Macbook
- OS version:
- Browser version: Chrome
- Voiceover SR

## 🧐 Expected behaviour

The contents of a dialog should be announced by the voiceover SR. It should read the dialog title and message before then announcing the focussed confirm button. 

## 📝 Acceptance Criteria

If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil.

_Given_ that any SR is being used
_When_  a dialog is launched
_Then_ its contents should be announced before any buttons are focussed. 

## Additional info

Tell us anything else useful to help us fix or understand the problem.
